app/src/pages/inside/logsPage/defectEditor/defectEditorHistoryLine/defectEditorHistoryLine.scss

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.history-line-container {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  max-width: 100%;
+  padding: 0 12px;
+  background-color: transparent;
+}
+
+.history-line-item {
+  margin-right: 8px;
+
+  & div:nth-child(2) {
+    cursor: default;
```

Comment:
Could you describe why we need this line?

Reply:
It removes cursor pointer from history line item